firstvoices/backend/serializers/fields.py

```diff
@@ -85,7 +85,7 @@ def to_internal_value(self, data):
     def to_representation(self, value):
```

Comment:
This is unrelated to this PR, but I'm looking at the to_internal_value method and it could be made simpler by skipping the visibility_map and just returning Visibility[data.upper()]